src/components/atomic/CopyButton.tsx

```diff
@@ -36,7 +37,7 @@ const CopyButton = ({ copyText, children }: CopyButtonProps) => {
   };
 
   return (
-    <button onClick={handleCopy} ref={buttonRef} className={`${isCopied ? 'cursor-default bg-green-700/20 text-green-700' : 'cursor-pointer text-brandDark dark:text-brandLight hover:opacity-80'} bg-brandDark/10 text-center font-medium py-1 flex items-center justify-center gap-2 px-4 whitespace-nowrap rounded-lg`}>
+    <button style={{ height: height }} onClick={handleCopy} ref={buttonRef} className={`${isCopied ? 'cursor-default bg-green-700/20 text-green-700' : 'cursor-pointer text-brandDark dark:text-brandLight hover:opacity-80'} bg-brandDark/10 text-center font-medium py-1 flex items-center justify-center gap-2 px-4 whitespace-nowrap rounded-lg`}>
```

Comment:
That won't work because Tailwind only generates its CSS file based on the classes it recognises when it scans the code. I would have to specifically tell Tailwind to always include these values in its config safelist. There is a package which does this automatically but I think that's excessive so just using the style tag is simplest.